test/tbb/test_collaborative_call_once.cpp

```diff
@@ -285,12 +285,12 @@ TEST_CASE("handles exceptions - state reset") {
 TEST_CASE("handles exceptions - stress test") {
 #if TBB_TEST_LOW_WORKLOAD
     constexpr std::size_t N = 32;
-#elif __TBB_x86_32 || __arm__ || __ANDROID__
+#elif SIZE_MAX <= 4294967295 // 32-bit targets
```

Comment:
The comment below says that Android was added to decrease testing time. It'd probably be better then to do:
(SIZE_MAX <= 4294967295) || __ANDROID__ // 32-bit targets or Android